tox.ini

```diff
@@ -20,6 +19,28 @@ whitelist_externals =
 commands =
     py.test -vv --durations=5
 
+# NOTE: Older version of tox which still supports Python 2.6 doesn't support installing
+# requirements from a file so we need to declare them inline
+# In addition to that, we need to use older versions of some of the libraries
+# (importlib, pytest, etc.)
+# We can get rid of this once we drop Python 2.6 support.
+[testenv:py2.6-unit-tests]
+basepython = python2.6
+install_command = pip install -U --force-reinstall {opts} {packages}
+deps =
```

Comment:
We need to make sure this stays in sync with the requirements file, right?  If so, how about adding documentation to it that if you modify the requirements, this should be updated as well.

Reply:
Yeah, my plan for a future PR was to improve that since right now the approach we use is not ideal (we shove test + dev requirements in a same file), in addition to that, those requirements are also copy and pasted in various places (Dockerfile, etc. and they are not even pinned to a specific / same version everywhere).

So my plan was to split ``dev-requirements.txt`` into multiple requirements file and then use those files as applicable (to avoid duplicating this data in multiple places).

If you think it belongs here, I can also add those changes here instead of working on them in a separate future PR.

Reply:
In the mean time, I went with a change so we read all dependencies for py2.6-unit-tests target from a file.